06-Tables/Tables.ipynb

```diff
@@ -11,32 +11,32 @@
     "- Handle tables containing missing values.\n",
     "- Include table and column metadata as flexible data structures.\n",
-    "- Specify a description, units and output formatting for columns.\n",
+    "- Specify a description, units, and output formatting for columns.\n",
     "- Perform operations like database joins, concatenation, and grouping.\n",
     "- Manipulate multidimensional columns.\n",
-    "- Methods for Reading and writing Table objects to files\n",
-    "- Integration with Astropy [Units and Quantities](http://astropy.readthedocs.org/en/stable/units/index.html)\n",
+    "- Methods for reading and writing Table objects to files.\n",
+    "- Integration with Astropy [Units and Quantities](http://astropy.readthedocs.org/en/stable/units/index.html).\n",
     "\n",
     "Tables vs. Pandas DataFrames\n",
-    "--------------\n",
+    "------------------------------------\n",
     "\n",
-    "The [Pandas](http://pandas.pydata.org/pandas-docs/stable/) package provides a powerful, high-performance table object via the [DataFrame](http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.html#pandas.DataFrame) class.  This object in many ways is similar to Astropy tables, and is rather broadly used in data science. If you have some familiarity with Pandas, you might wonder why it is not used instead of an astronomy-specific solution.\n",
+    "The [Pandas](http://pandas.pydata.org/pandas-docs/stable/) package provides a powerful, high-performance table object via the [DataFrame](http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.html#pandas.DataFrame) class.  This object is similar to Astropy Table in many ways and is rather broadly used in data science. If you have some familiarity with Pandas, you might wonder why it is not used instead of an astronomy-specific solution.\n",
     "\n",
-    "While here are a few shortcomings that prevent its use as a generalized table object in astronomy.  These reasons include:\n",
-    "* Pandas does not support table \"columns\" that are more complicated than simple numbers. Astropy tables do, enabling all of the features of Astropy objects to be used inside Tables. \n",
-    "* Pandas does not support for multidimensional table columns.  This is commonly used in standard FITS data products, for instance the Chandra energy response matrix used to analyze spectral data.  \n",
-    "* Pandas' support for missing data relies on NaNs instead of masks.  For some astronomical applications this is not desired, as it deletes potentially useful information and makes some standard astronomical analysis tasks overly complicated.\n",
+    "There are a few reasons why Pandas is not used as a generalized table object in astronomy. These reasons include:\n",
+    "* Pandas does not support table \"columns\" that are more complicated than simple numbers. Astropy Table does, enabling all of the features of Astropy objects to be used inside tables. \n",
```

Comment:
I think here it would make some sense to markdown around `Table` or keep it as generic as Astropy tables (. 
The end of line however should stay capital Table or Tables. I'm not sure whether it's covered in our guidelines, my rule of thumb it's lowercase when generically speaking about astropy tables, but definitely capitalized when talking about the objects)

Reply:
Thanks @bsipocz! Sometimes I'm confused as to whether it's referring to generic tables or the Table object, so thanks for catching this!